usagov/usagov-2021
# USAGOV-1532-html-tag-limit: configuration to allow html

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1532

## Description

Turn on the "Limit allowed HTML tags and correct faulty HTML" filter
## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Import updated configuration with "bin/drush cim"
- Login into your local cms and select the "Configuration" tab in the top bar
- Select "Text formats and editors"
- <img width="1512" alt="Screenshot 2024-06-14 at 1 21 40 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/cf9d83ba-349e-4a64-9b5a-727e2b9626a5">

- Then configure the USWDS block
- <img width="1445" alt="Screenshot 2024-06-14 at 1 21 53 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/32d47469-1340-4801-a0d5-e6250a4b2ea6">

- Then select "Source editing" and verify the html box contains "<div>" inside and verify the "Limit allowed HTML tags and correct faulty HTML" is selected
- <img width="1512" alt="Screenshot 2024-06-14 at 1 22 16 PM" src="https://github.com/usagov/usagov-2021/assets/5551880/d49c9448-19f4-4fb5-8a02-22c80ddc7acc">


## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
